python/lsst/meas/base/applyApCorr.py

```diff
@@ -194,11 +194,11 @@ def run(self, catalog, apCorrMap):
                 if self.config.doFlagApCorrFailures:
                     source.set(apCorrInfo.fluxFlagKey, oldFluxFlagState)
 
-            if self.log.getThreshold() <= self.log.DEBUG:
+            if self.log.getLevel() <= self.log.TRACE:
```

Comment:
You are right. I changed it back to DEBUG. 

Also squashed the two commits into one so it's clearer to see this.